### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by henry-richard**

### CODE REVIEW

**Comment by chad-daniels** [comment]
looks good

**Comment by chad-daniels** [comment]
should i merge?

File: README.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ test
 
 smoke test p2
 test
```
Comment (wanda-carlson) [inline README.md:8]: comment 1

File: README.md [hunk 2]
```diff
@@ -6,3 +6,4 @@ test
...
+test webhook-secret
```
Comment (wanda-carlson) [inline README.md:9]: comment 2 @henry-richard

**Review by wanda-carlson — CHANGES_REQUESTED** [review]

**Review by wanda-carlson — COMMENTED** [review]
please add the changes

**Review by wanda-carlson — COMMENTED** [review]
testingggg

**Comment by wanda-carlson** [comment]
another one

File: README.md (hunk 1 above)
Comment (henry-richard) [inline README.md:8]:
```suggestion
test 123
```

**Review by henry-richard — COMMENTED** [review]
test suggestion

**Pull request closed without merging**